androidHackathonApp/src/main/res/menu/bottom_nav_menu.xml

```diff
@@ -7,6 +7,11 @@
         android:icon="@drawable/ic_baseline_map_24"
         android:title="@string/map_tab_label" />
 
+    <item
+        android:id="@+id/addMealFragment"
+        android:icon="@drawable/ic_baseline_add_24"
+        android:title="Add a meal" />
```

Comment:
We generally try to always put text in the `strings.xml` file and avoid hardcoding it.